packages/atomic-layout/src/hooks/useMediaQuery.ts

```diff
@@ -66,3 +66,5 @@ export const useMediaQuery: UseMediaQuery = (
 
   return matches
 }
+
+export default useMediaQuery
```

Comment:
Why to change it to default export?
I'd try to avoid using default export from now one due to [this motivation](https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html). If you could, please, revert it back to the named export and use `export { useMediaQuery } from '...'` in the entry module. Thanks!

Reply:
done that, and thanks for the link,